backend/app/models/inquiry.py

```diff
@@ -1,27 +1,33 @@
 import uuid
 from datetime import datetime, timezone
+from typing import List, Optional
 
-from sqlmodel import Field, SQLModel
+from sqlmodel import Field, Relationship, SQLModel
+
+
+class Response(SQLModel, table=True):
```

Comment:
Like I mentioned above, we could maybe stub out the Response for this ticket. If you want to get the skeleton for the Response model coded up, it should not live in this file. It should be in its own module.